spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -137,16 +136,18 @@ public class Student {
 
 In this example, the `name` field is annotated with `@NotBlank`, indicating that it must not be blank. The `age` field is annotated with both `@Positive` and `@Min(18)`, ensuring it is a positive integer greater than or equal to 18.
 
-The `School` repository interface, shown below, utilizes the validation rules defined in the `Student` entity:
+The `School` repository interface, shown below, uses the `jakarta.validation.Valid` annotation to cause the constraints from the `Student` entity to be validated during the `save` operation, whereas the validation constraints are not applied to the `Student` entities returned as a result of the `findByAgeLessThanEqual` operation because the `findByAgeLessThanEqual` method does not include a `jakarta.validation.Valid` annotation that applies to the return value.
 
 [source,java]
 ----
 @Repository
-public interface School extends PageableRepository<Student, String> {
+public interface School extends DataRepository<Student, String> {
```

Comment:
```suggestion
public interface School extends DataRepository<@jakarta.validation.Valid Student, String> {
```
The way that Micronaut does sounds more semantic:

https://micronaut-projects.github.io/micronaut-data/latest/guide/

I mean, overwrite the method to include the valid annotation sounds noise in the method.

Reply:
> The way that Micronaut does sounds more semantic:

@otaviojava  I like the approach that Micronaut has as well.  But Jakarta Validation doesn't currently allow that pattern.  One of the Micronaut developers and myself have been in contact with the Jakarta Validation team about it, and I've opened issue https://github.com/jakartaee/validation/issues/186 officially requesting them to add this ability for Jakarta EE 11.  If they add it, we'll update our spec documentation to favor that approach.  But for now, our specification is limited to what is currently supported by Jakarta Validation.